### DISCUSSION TITLE
Version v1.0.0 Released

### DISCUSSION THREAD

**Discussion by hollowaykeanho** [ZORALab/demo-web-prestige#27] [Announcements]
Just released after the 2-days power-on with UI developed from scratch and based on prestige minimalistic style design. Please do check it out at: https://prestige-demo.zoralab.com/

**Comment by corygalyna** [comment]
Finally!!! Great lord. Took us the nights. =)

**Comment by corygalyna** [comment]
Pitch to customer first?

**Comment by hollowaykeanho** [comment]
already doing it.
